volume-group-operator/controllers/utils/volumegroup.go

```diff
@@ -46,3 +67,67 @@ func UpdateVolumeGroupStatusError(client client.Client, instance *volumegroupv1.
 
 	return nil
 }
+
+func GetVGList(logger logr.Logger, client client.Client) (volumegroupv1.VolumeGroupList, error) {
+	logger.Info(messages.ListVolumeGroups)
+	vg := &volumegroupv1.VolumeGroupList{}
+	err := client.List(context.TODO(), vg)
```

Comment:
is there a way to filter here by PVC label and volume group selector?

Reply:
I didn't find a way, you usually filter objects with labelSelector and the PVC doesn't have labelSelector, so I didn't find a way to get only the VGs that match the PVC in one command, I can open a ticket for after the GA to investigate it, but I didn't find a way when I did it